metricbeat/module/kubernetes/state_persistentvolumeclaim/state_persistentvolumeclaim_test.go

```diff
@@ -28,6 +28,14 @@ import (
 func TestEventMapping(t *testing.T) {
 	ptest.TestMetricSet(t, "kubernetes", "state_persistentvolumeclaim",
 		ptest.TestCases{
+			{
+				MetricsFile:  "../_meta/test/ksm.v1.3.0",
+				ExpectedFile: "./_meta/test/ksm.v1.3.0.expected",
+			},
+			{
+				MetricsFile:  "../_meta/test/ksm.v1.8.0",
+				ExpectedFile: "./_meta/test/ksm.v1.8.0.expected",
+			},
 			{
 				MetricsFile:  "./_meta/test/ksm-unit-1.8",
```

Comment:
Would it be of benefit to keep some consistency here regarding the naming?:
```suggestion
				MetricsFile:  "./_meta/test/ksm.unit.v1.8.0",
```

Reply:
sounds good